lib/engine/game/g_1846.rb

```diff
@@ -391,6 +391,31 @@ def sellable_bundles(player, corporation)
         super
       end
 
+      def issuable_shares(entity)
+        return [] unless entity.corporation?
+        return [] unless round.steps.find { |step| step.class == Step::G1846::Dividend }.active?
+
```

Comment:
I don't like inferring that the player can issue and redeem as long as the Dividend step is still active in multiple places. I think it would be clearer if it were instead to check if the IssueShares step is active; however, right now the code never skips that step and instead returns no actions if the Dividend step isn't active. Is it safe in IssueShares::actions() to call skip!? That would allow me to check for the IssueShares step being active here.

Reply:
probably better to call step.pass!

Reply:
That's what I meant. :-)